app/services/idv/session.rb

```diff
@@ -66,21 +65,18 @@ def create_profile_from_applicant_with_password(user_password)
 
       profile.activate unless profile.reason_not_to_activate
 
-      self.pii = profile_maker.pii_attributes
```

Comment:
As far as I can tell this acts like an ivar, but since this object represents a session it resulted in the PII actually being written to the session. Thankfully the entirety of the idv session is encrypted so this PII is encrypted.